model/distro/distro_test.go

```diff
@@ -428,3 +429,31 @@ func TestGetResolvedPlannerSettings(t *testing.T) {
 	assert.EqualValues(t, 0, resolved2.TimeInQueueFactor)
 	assert.EqualValues(t, 0, resolved2.ExpectedRuntimeFactor)
 }
+
+func TestAddPermissions(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
I would just pass t to all the asserts, also perhaps break this case up with sub tests so that it'll fail more clearly in the future.